packages/components/src/ConditionalFilter/GroupFilter.js

```diff
@@ -267,21 +309,37 @@ class Group extends Component {
                         groups.map(({
                             value: groupValue,
                             onSelect,
+                            groupSelectable,
                             label: groupLabel,
                             id: groupId,
                             type,
                             items,
                             ...group
                         }, groupKey) => {
-                            const filteredItems = this.mapItems({ groupValue, onSelect, groupLabel, groupId, type, items, ...group }, groupKey)
+                            const filteredItems = this.mapItems({ groupValue, onSelect, groupLabel, groupId, groupSelectable, type, items, ...group }, groupKey)
                             .filter(Boolean);
-                            return (<SelectGroup
-                                {...group}
-                                key={groupId || groupValue || groupKey}
-                                value={groupId || groupValue || groupKey}
-                                label={groupLabel || ''}
-                                id={groupId || `group-${groupValue || groupKey}`}
-                            >{filteredItems}</SelectGroup>);
+
+                            return (
+                                <div key={groupId || groupValue || groupKey}>
+                                    {
+                                        groupSelectable && <SelectOption
+                                            onClick={ (event) => {
+                                                this.onGroupSelect(event, groupValue || groupKey, items);
+                                            } }>
+                                            <Checkbox
+                                                isChecked={ this.isGroupChecked(groupValue || groupKey, items) }
+                                                label={ groupLabel } />
+                                        </SelectOption>
+                                    }
+                                    <SelectGroup
+                                        {...group}
+                                        className='pf-u-pl-sm'
+                                        label={ !groupSelectable && groupLabel }
```

Comment:
I'm kinda sceprical about this field. PF has hard check for `props.label.replace` without safe accessors. We've had problems with it in the past. The worst thing is that you can't have nodes as label (so no translations or the select option from above in here).